src/main/java/com/scalar/admin/AdminCommand.java

```diff
@@ -107,6 +111,9 @@ public Integer call() {
       throw new IllegalArgumentException(
           "It's required to specify only either [--srv-service-url, -s] or [--addresses, -a].");
     } else if (srvServiceUrl != null) {
+      logger.error(
```

Comment:
```suggestion
      logger.warn(
```
Why do you use `error`?
Also, I don't think we need `Warnings:` header since it's done by the logger.

Reply:
@feeblefakie 

Sorry, it was my typo. It should be `warn`.

(I remembered I used `warn` though, anyway.)

> Also, I don't think we need Warnings: header since it's done by the logger.

Ah, I see. I did this because I configured the log4j2 output pattern to be just plaintext.

I made a change according to this comment at https://github.com/scalar-labs/scalar-admin/pull/51/commits/27315b4d3fb8fef5e50aa7ac8fcb1316b3dc51bb

Please take a look 🙇